WordPress/Classes/ViewRelated/NUX/WPWalkthroughTextField.m

```diff
@@ -57,8 +59,8 @@ - (void)commonInit
     self.showTopLineSeparator = NO;
     self.showSecureTextEntryToggle = NO;
 
-    self.secureTextEntryImageVisible = [UIImage imageNamed:WPIconShowPassword];
-    self.secureTextEntryImageHidden = [UIImage imageNamed:WPIconHidePassword];
+    self.secureTextEntryImageVisible = [Gridicon iconOfType:GridiconTypeVisible];
+    self.secureTextEntryImageHidden = [Gridicon iconOfType:GridiconTypeNotVisible];
 
     self.secureTextEntryToggle = [UIButton buttonWithType:UIButtonTypeCustom];
     self.secureTextEntryToggle.frame = CGRectMake(0, 0, 40, 30);
```

Comment:
Considering this is outside the scope of the PR, can you specify the change you'd like here?  I'd like to avoid a trial and error.

If I adjust the button to the default Gridicon size, the new code is going to have a different size.

PS: I don't think it'd be wise to continue extending this PR, but I don't mind changing it if the request is specific.